api/controllers/install/controller.go

```diff
@@ -31,6 +31,10 @@ type Controller interface {
 	GetStatus(ctx context.Context) (*types.Status, error)
 }
 
+type RunHostPreflightsOptions struct {
```

Comment:
might be better to move this to a types package

Reply:
I think we should discuss the types package. Maybe break it up. I had the understanding that it was for api types (eg request response), not controller and manager options.

https://github.com/replicatedhq/embedded-cluster/pull/2300/files#diff-f62a0fad1ad21e2ffa9cf4de2cab02d8a2dccd2557065c737942d54e9a1ed736R3